rackcity/views/it_model_views.py

```diff
@@ -83,6 +93,24 @@ def model_modify(request):
     )
 
 
+def model_height_change_valid(new_model_data, existing_model):
+    if 'height' not in new_model_data:
+        return True
+    if new_model_data['height'] <= existing_model.height:
+        return True
+    else:
+        instances = ITInstance.objects.filter(model=existing_model.id)
+        for instance in instances:
+            if is_location_full(
+                instance.rack,
```

Comment:
addressed this in the original PR, but make sure this is `instance.rack.id` when you merge

Reply:
Weird it seems to work either way but you are correct